tests/unit/classifier/svm/SVMLight_unittest.cc

```diff
@@ -0,0 +1,53 @@
+#include <gtest/gtest.h>
+
+#include "environments/GaussianCheckerboard.h"
+#include <shogun/classifier/svm/SVMLight.h>
+#include <shogun/evaluation/ContingencyTableEvaluation.h>
+#include <shogun/kernel/GaussianKernel.h>
+#include <shogun/lib/SGVector.h>
+#include <shogun/lib/config.h>
+
+using namespace shogun;
+
+#ifdef USE_SVMLIGHT
+TEST(SVMLight, CacheSizeInvariance)
+{
+	const int32_t num_samples = 50;
```

Comment:
Minor on style, uniform initialization is preferred. No need to update it here; I find it a convenient rule.